Source/WebCore/rendering/RenderLayerModelObject.cpp

```diff
@@ -291,6 +291,35 @@ void RenderLayerModelObject::updateLayerTransform()
         layer()->updateTransform();
 }
 
+bool RenderLayerModelObject::setReferenceBoxForPathOperations()
+{
+    auto pathOperation = style().offsetPath();
+    if (!pathOperation)
+        return true;
+    if (is<BoxPathOperation>(pathOperation)) {
+        auto& boxPathOperation = downcast<BoxPathOperation>(*pathOperation);
+        auto pathReferenceBoxRect = snapRectToDevicePixelsIfNeeded(referenceBoxRect(boxPathOperation.referenceBox()), *this);
+        boxPathOperation.setPathForReferenceRect(FloatRoundedRect { pathReferenceBoxRect });
```

Comment:
You could `return` here and remove the `else`

Reply:
Fixed.